src/sentry/api/endpoints/organization_invite_request_details.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import absolute_import
+
+from django.db.models import Q
+from django.conf import settings
+from rest_framework import serializers
+from rest_framework.response import Response
+
+from sentry import roles
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize, OrganizationMemberWithTeamsSerializer
+from sentry.api.serializers.rest_framework import ListField
+from sentry.models import AuditLogEntryEvent, InviteStatus, OrganizationMember, Team, TeamStatus
+from sentry.signals import member_invited
+
+from .organization_member_index import save_team_assignments
+
+
+class InviteRequestSerializer(serializers.Serializer):
```

Comment:
Can we reuse the other validator here somehow? Ideally not subclassing.

Reply:
updated